### PR TITLE
updated launcher for easy launching of modules and validators

### PR DESCRIPTION
**Pull request by Bakobiibizo**
modified the synthia launch to easily launch miners and validators

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,47 @@ We are initiating our subnet by deploying simple embedding miners. These miners
...
+
 ## Embedding Miner
 
 Base miner is in the [miner](eden_subnet/miner/miner.py) folder. Copy the class `Miner` and pass it the `MinerSettings` class to deploy your own version. The repo references the position of the miner class and the file it resides in through a naming convention of the miner. Use <FILE_NAME>.<CLASS_NAME> for both the name of the key you are staking with and the file/class name of the code to your miner. 
-To launch your miner run 
-`python -m eden_subnet.cli serve-miner miner.Miner miner.Miner 0.0.0.0 10001`
-
+To launch your miner you need to open a new python script in `eden_subnet/miner` and inherit the miner with a new class. You can see an example of this [here](eden_subnet/miner/eden.py)
+```#python
+from edne_subnet.miner.miner import Miner, MinerSettings
```
Comment (coolrazor007) [inline README.md]: is the spelling error here a blocker?  edne_subnet

File: README.md [hunk 2]
```diff
@@ -19,12 +19,47 @@ We are initiating our subnet by deploying simple embedding miners. These miners
 **VRAM**: min 8gb
 Internet connection
 
+## Launcher
+
+The easiest way to launch miners and validators is using the launcher.sh script if you're on linux.
+
+`chmod -x launcher.sh`
```
Comment (coolrazor007) [inline README.md]: this should be +x right?  -x removes execute

**Comment by coolrazor007** [comment]
I think in the requirements or somewhere you should add "pip install -U communex"

**Comment by coolrazor007** [comment]
I ran the launcher.sh and chose option 2 but it failed to launch
image

**Comment by coolrazor007** [comment]
> I ran the launcher.sh and chose option 2 but it failed to launch image

this was me being dumb.  My Python3 wasn't responding to Python requests

**Comment by Bakobiibizo** [comment]
> I think in the requirements or somewhere you should add "pip install -U communex"
its in the pyproejct.toml and in the requirements.txt

**Review by coolrazor007 — APPROVED** [review]

**Pull request merged**